modules/ServerAPI/src/server/routes/ServerUtilityFunctions.coffee

```diff
@@ -230,7 +230,7 @@ exports.getFromACASServerInternal = (baseurl, callback) ->
 		json: true
 	, (error, response, json) =>
 		if !error && response.statusCode == 200
-			callback response.statusCode, JSON.stringify json
```

Comment:
This change is going to break other modules. Need to revert it and add in an alternate route before merging this into anywhere